### ISSUE TITLE
Nested content clipboard broken after upgrade to Umbraco 8.6.2

### ISSUE THREAD

**Issue by creativesuspects** [umbraco/Umbraco-CMS#8223]
A brief description of the issue goes here.

After upgrading to Umbraco 8.6.2 it's no longer possible to copy nested content to the clipboard. Both the 'copy all' and 'copy single' functions are broken.

## Umbraco version

I am seeing this issue on Umbraco version: 8.6.2.
I've also tested it on a clean install, same results.

### Bug summary

I'm getting the following reference error:

```
ReferenceError: Utilities is not defined
    at prepareEntryForStorage (umbraco.services.js?cdv=1:917)
    at Object.service.copy (umbraco.services.js?cdv=1:982)
    at NestedContentController.vm.clickCopy (umbraco.controllers.js?cdv=1:21120)
    at fn (eval at compile (angular.js?cdv=1:16421), <anonymous>:4:310)
    at callback (angular.js?cdv=1:28954)
    at ChildScope.$eval (angular.js?cdv=1:19396)
    at ChildScope.$apply (angular.js?cdv=1:19495)
    at HTMLButtonElement.<anonymous> (angular.js?cdv=1:28958)
    at HTMLButtonElement.dispatch (jquery.min.js?cdv=1:2)
    at HTMLButtonElement.v.handle (jquery.min.js?cdv=1:2)
```

This is caused by line 917 in /Umbraco/Js/umbraco.services.js:

`var cloneData = Utilities.copy(entryData);`

### Specifics

This seems to be related to the fix for issue #7133.

### Steps to reproduce

![Recording #40](https://user-images.githubusercontent.com/161342/83549803-29ece800-a506-11ea-900a-e7107a65248a.gif)

**Comment by creativesuspects** [comment]
It looks like /src/Umbraco.Web.UI.Client/src/utilities.js is somehow not included in umbraco.services.js even though it should be according to the /src/Umbraco.Web.UI.Client/gulp/config.js line 37:

`services: { files: ["./src/common/services/**/*.js", "./src/utilities.js"], out: "umbraco.services.js" },`

When I manually add the contents of /src/Umbraco.Web.UI.Client/src/utilities.js to the bottom of umbraco.services.js the clipboard function works again.

**Comment by nul800sebastiaan** [comment]
Oh no! 

For now this can be manually fixed by replacing line 917 in `\Umbraco\Js\umbraco.services.js` from `var cloneData = Utilities.copy(entryData);` to `var cloneData = angular.copy(entryData);` and then bumping ClientDependency version (just make it 1 higher or lower).

This PR should fix it, error was in using a js class that doesn't yet exist in 8.6 - https://github.com/umbraco/Umbraco-CMS/pull/8226

**Comment by clausjensen** [comment]
Fixed by #8226
